src/main/app/case/from-api-format.ts

```diff
@@ -5,16 +5,16 @@ import { Case, Checkbox, YesOrNo, formFieldsToCaseMapping, formatCase } from './
 
 const fields = {
   ...invert(formFieldsToCaseMapping),
-  D8MarriageIsSameSexCouple: data => ({
+  D8MarriageIsSameSexCouple: (data: Partial<CaseData>): Partial<Case> => ({
```

Comment:
I'd rather not have the type definitions on this as it makes it very bulky. Is it necessary?

Reply:
Technically it's not strictly necessary. I was using it to get the correct type checking, hinting and autocomplete.

Reply:
I've remove these in 0f6aed0